### PR TITLE
[SDPA-1729] Adds wrapper for vueMultiselect to use with object arrays.

### PR DESCRIPTION
**Pull request by dylankelly**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-1729

### Changed

1.  Adds fieldRplSelect VFG component which wraps vuemultiselect component to handle arrays of objects. Currently only supports single selection.
1.  Fixes required indicator propagating to child labels

### CODE REVIEW

File: src/storybook-components/_data/demoData.js [hunk 1]
```diff
@@ -1323,7 +1323,8 @@ const demoData = {
         dateRange: ['', ''],
         date: '',
         checkbox: true,
-        vuemultiselect: null,
+        select: [],
```
Comment (alan-cole) [inline src/storybook-components/_data/demoData.js]: Is this new select model used anywhere?

File: packages/Molecules/Form/index.vue [hunk 2]
```diff
@@ -173,7 +173,10 @@ export default {
 
   .error {
     input,
-    textarea {
+    textarea,
+    .multiselect__tags,
+    .form-control
+    {
```
Comment (alan-cole) [inline packages/Molecules/Form/index.vue]: If the SCSS linting worked in .vue files, I think this one would be picked up. { should be on the same line as the last rule.

File: packages/Molecules/Form/stories.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ storiesOf('Molecules/Form', module)
   .addDecorator(VueInfoAddon)
   .add('Form', withReadme(readme, () => ({
     components: { RplForm },
-    template: `<rpl-form :formData="formData"></rpl-form>`,
+    template: `<rpl-form :formData="formData" :submitHandler="submitHandler" :is-new-model="isNewModel" :options="options"></rpl-form>`,
```
Comment (alan-cole) [inline packages/Molecules/Form/stories.js]: Should there be some consistency between prop naming, should `:is-new-model` be `:isNewModel`?

File: packages/Molecules/Form/index.vue (hunk 2 above)
Comment (dylankelly) [reply packages/Molecules/Form/index.vue]: thanks, need to fix that linter!

**Pull request merged**